### PR TITLE
PROXY-9: Refactor code / Add parameters to the query 

### PR DESCRIPTION
**Pull request by haradzetskaya**

### CODE REVIEW

File: src/services/util-service.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { format } from 'date-fns';
+
+const pattern = 'yyyy-MM-dd';
```
Comment (MShachykau) [inline src/services/util-service.js:3]: I think would be great if you create the constant.js file and will export this constant from that file. 
ex. `export const FORMAT_DATE = 'yyyy-MM-dd'`

File: src/services/util-service.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+export const formatDate = (date) => {
+    return format(date, pattern);
```
Comment (MShachykau) [inline src/services/util-service.js:6]: it looks unnecessary, we can just call the `format` function from `date-fns` where we need it.

File: src/controllers/asteroid-controller.js [hunk 3]
```diff
@@ -41,5 +48,19 @@ export const getMeteorites = asyncHandler (async (req, res, next) => {
...
+    if (count) {
+        const { element_count } = data;
+        total = element_count;
+    }
+
+    let is_dangerous;
+    if (were_dangerous_meteors) {
+        let dangerous_asteroids = asteroids.filter((meteor) => {
+            return meteor.is_potentially_hazardous_asteroid === true
+        })
```
Comment (MShachykau) [inline src/controllers/asteroid-controller.js:61]: you can you this construction here 
`asteroids.filter(({is_potentially_hazardous_asteroid}) => is_potentially_hazardous_asteroid === true)`.

File: src/services/util-service.js (hunk 2 above)
Comment (haradzetskaya) [reply src/services/util-service.js]: done

File: src/controllers/asteroid-controller.js (hunk 3 above)
Comment (haradzetskaya) [reply src/controllers/asteroid-controller.js]: done

**Pull request merged**